### PR TITLE
docs/variants/msi_z790: add initial documentation

### PR DESCRIPTION
**Pull request by miczyg1**

### CODE REVIEW

File: docs/variants/msi_z790/faq.md [hunk 1]
```diff
@@ -0,0 +1,164 @@
...
+* By buying this product, you support open-source firmware and Dasharo
+  distribution.
+
+## What means in warranty conditions "No signs of customer interference..."?
+
+Full question:
+"In the Disclaimer, you wrote that one of the warranty
+conditions is: "No signs of customer interference with the platform or
+firmware." How would a user modify firmware without losing the warranty? Being
+open source to be easy to modify, then saying "don't touch this" seems
```
Comment (mkopec) [inline docs/variants/msi_z790/faq.md:158]:
```suggestion
open-source to be easy to modify, then saying "don't touch this" seems
```

File: docs/variants/msi_z790/firmware-update.md [hunk 2]
```diff
@@ -0,0 +1,130 @@
+# Firmware update
+
+The following documentation describes the process of Dasharo open-source
+firmware update. The update process may be different, depending on which
+firmware version is currently installed on your device. The currently installed
+firmware version can be checked with the following command in a Linux
+environment:
```
Comment (mkopec) [inline docs/variants/msi_z790/firmware-update.md:7]:
```suggestion
The following documentation describes the process of updating Dasharo
open-source firmware. The update process may be different depending on which
firmware version is currently installed on your device. The currently installed
firmware version can be checked with the following command in a Linux
environment:
```

**Review by mkopec — CHANGES_REQUESTED** [review]

File: docs/variants/msi_z790/recovery.md [hunk 3]
```diff
@@ -0,0 +1,252 @@
...
+  may choose an older one too or in the best case use your firmware backup):
+
+```bash
+<<<<<<< HEAD
+wget https://download.msi.com/bos_exe/mb/7D25v13.zip
+unzip 7D25v13.zip
+=======
+wget https://download.msi.com/bos_exe/mb/7E06vA3.zip
+unzip 7E06vA3.zip
+>>>>>>> dc0451a6 (docs/variants/msi_z790: add initial documentation)
```
Comment (mkopec) [inline docs/variants/msi_z790/recovery.md:35]: please fix

File: docs/variants/msi_z790/recovery.md [hunk 4]
```diff
@@ -0,0 +1,252 @@
...
+* Flash via external programmer:
+
+> The command line will be different, depending on the programmer you use
+
+```bash
+<<<<<<< HEAD
+flashrom -p linux_spi:dev=/dev/spidev1.0,spispeed=16000 -w 7D25v13/E7D25IMS.130
+=======
+flashrom -p linux_spi:dev=/dev/spidev1.0,spispeed=16000 -w 7E06vA3/E7E06IMS.A30
+>>>>>>> dc0451a6 (docs/variants/msi_z790: add initial documentation)
```
Comment (mkopec) [inline docs/variants/msi_z790/recovery.md:47]: please fix

**Comment by mkopec** [comment]
I tried to simplify the language where I could, to make it easier to understand for non-technical people and non-native English speakers.

**Comment by mkopec** [comment]
@miczyg1 pre-commit is failing here

File: docs/variants/msi_z790/initial-deployment.md [hunk 5]
```diff
@@ -0,0 +1,128 @@
+# Initial Deployment
+
+<<<<<<< HEAD
+Initial deployment of Dasharo firmware on MSI PRO Z690-A can be done:
+=======
+Initial deployment of Dasharo firmware on MSI PRO Z790-P can be done:
+>>>>>>> dc0451a6 (docs/variants/msi_z790: add initial documentation)
```
[the remark this answers is not in the document]
Comment (miczyg1) [reply docs/variants/msi_z790/initial-deployment.md]: Fixed by force pushing to 544d914b198e28f21f2d8ef088f2425d656c0836

File: docs/variants/msi_z790/recovery.md (hunk 4 above)
Comment (miczyg1) [reply docs/variants/msi_z790/recovery.md]: Fixed by force pushing to 544d914b198e28f21f2d8ef088f2425d656c0836

File: docs/variants/msi_z790/faq.md [hunk 6]
```diff
@@ -0,0 +1,164 @@
+# FAQ
+
+## Why MSI Z790-P (WIFI) (DDR4) was chosen for Dasharo ?
+
+Please read [this extensive analysis][msi_port_analysis] contributed to Intel
+Reddit.
```
[the remark this answers is not in the document]
Comment (miczyg1) [reply docs/variants/msi_z790/faq.md:6]: why not `r/Intel`?

**Comment by miczyg1** [comment]
> If those platforms are so similar, then why will we not handle differences in a similar way to NovaCustom? @macpijan ?

I wanted to do some unification as a next step.

**Comment by macpijan** [comment]
> If those platforms are so similar, then why will we not handle differences in a similar way to NovaCustom?

We should do that.

**Comment by miczyg1** [comment]
@mkopec pre-commit no longer failing. Can I get the approve? I will start unifying the documentation then

**Comment by miczyg1** [comment]
@mkopec ping

**Review by mkopec — APPROVED** [review]

**Pull request merged**